packages/example/src/screens/HeadlessCheckoutWithRedirect.tsx

```diff
@@ -153,69 +201,13 @@ const HeadlessCheckoutWithRedirect = (props: any) => {
     </TouchableOpacity>
   );
 
-  const renderBanks = () => {
-    // return (
-    //   <FlatList
-    //     data={banks}
-    //     renderItem={({ item }) => <Bank item={item} />}
-    //     keyExtractor={item => item.id}
-    //   />
-    // );
-
-    if (banks.length != 0) {
-      return banks.map((bank: IBank) => {
-
-        return <Bank key={bank.id} item={bank} />
-      })
-    }
-  };
-
-  const searchBanks = async (value: string) => {
-    setSearch(value);
-
-    try {
-      await redirectManager.onBankFilterChange(value);
-    } catch (err) {
-      console.error(err);
-    }
-  };
-
-  const renderSearch = () => {
-    return (
-      <TextField
-        title="Choose your bank"
-        textInputStyle={{
-          backgroundColor: '#f5f5f5',
-          borderColor: '#f5f5f5',
-          paddingHorizontal: 10,
-        }}
-        style={{
-          padding: 5,
-          marginVertical: 10,
-          borderRadius: 5,
-          marginHorizontal: 5,
-        }}
-        placeholder="Search Banks"
-        value={search}
-        onChangeText={text => {
-          searchBanks(text);
-        }}
-      />
-    );
-  };
+  if (banks.length != 0) {
+    return banks.map((bank: IBank) => {
+      return <Bank key={bank.id} item={bank} />
+    })
+  }
```

Comment:
[feedback] by inverting the condition, like a guard clause, we can make the return statement much clearer and make the intermediate component `Bank` obsolete, e.g.:

```ts
const Banks = ({ banks, isValidating, onPay }) => {
  if (!banks.length) return null;

  return banks.map((bank) =>
    <TouchableOpacity>
      // ...
    </TouchableOpacity>
  );
};
```

Alternatively, we could also simply use an optional chaining operator `?.`, e.g.:

```ts
const Banks = ({ banks, isValidating, onPay }) => {
  return banks?.map((bank) =>
    <TouchableOpacity>
      // ...
    </TouchableOpacity>
  );
};
```

Either way, if you'd like to keep the `Bank` component, please also create it outside `Banks`, otherwise the function is recreated on every render.